packages/components/src/styles/table-cell/table-cell.css

```diff
@@ -5,6 +5,56 @@
     @apply border-r-[1px];
   }
 
+  &--shadow-bottom {
+    &:after {
+      @apply opacity-0 absolute pointer-events-none transition-opacity duration-1000 ease-out	left-0 right-0 h-2.5 bottom-[-10px] content-[''] bg-gradient-to-b from-neutral-300 to-transparent;
```

Comment:
why does this shadow have a different transition timing function (ease-out) than all other shadows?

Reply:
That shouldn't be the case, it's a remnant of development.